router-tests/integration_test.go

```diff
@@ -52,10 +52,11 @@ func TestSimpleQuery(t *testing.T) {
 }
 
 func TestConfigReload(t *testing.T) {
-	t.Parallel()
 
 	t.Run("Successfully reloads to a valid new configuration file with SIGHUP", func(t *testing.T) {
 		// Can be very slow, compiles the router binary if needed
+
+		t.Setenv("CONFIG_PATH", "demo2.config.yaml")
```

Comment:
Please create a separate test and communicate that these tests can't be run in parallel.